controllers/disruption_controller.go

```diff
@@ -493,7 +500,14 @@ func (r *DisruptionReconciler) createChaosPods(instance *chaosv1beta1.Disruption
 		// create injection pods if none have been found
 		switch len(found) {
 		case 0:
-			r.log.Infow("creating chaos pod", "target", target)
+			chaosPodArgs := []string{}
+			if len(chaosPod.Spec.Containers) > 0 && chaosPod.Spec.Containers[0].Name == "injector" {
```

Comment:
i'm fairly confident this will always be true, but wanted to be safe.

Reply:
please consider to log something here. I feel there are opportunities to simplify  both the wrapping for loop (`	for _, chaosPod := range targetChaosPods {`) and this `case 0` condition